qualtran/bloqs/data_loading/select_swap_qrom_test.py

```diff
@@ -31,15 +31,14 @@
 from qualtran.testing import assert_valid_bloq_decomposition
 
 
-@pytest.mark.slow
 @pytest.mark.parametrize(
     "data,block_size",
     [
         pytest.param(
             data,
             block_size,
             id=f"{block_size}-data{didx}",
-            marks=pytest.mark.slow if block_size == 2 and didx == 0 else (),
+            marks=pytest.mark.slow if block_size == 1 and didx == 1 else (),
```

Comment:
This is probably also not slow ?

Reply:
It's >1s on my machine, but less than my personal 2s threshold for marking... but the other configurations should provide enough coverage for the `-m 'not slow'` executions of the unit tests.